vaadin-tab.html

```diff
@@ -18,7 +18,15 @@
         opacity: 0.7;
         padding: 0.3em 1em;
         box-sizing: border-box;
-        flex-shrink: 0;
```

Comment:
Should keep `flex-shrink` to avoid wrapping text in the tabs, as seen in the updated screenshots. The screenshots should not change.